libs/constants/index.js

```diff
@@ -124,6 +124,7 @@ const CSP_DIRECTIVES = {
     "*.githubusercontent.com",
     "*.googleusercontent.com",
     "mozillausercontent.com",
+    "https://firefoxusercontent.com/",
```

Comment:
```suggestion
    "firefoxusercontent.com",
```
let's stick to just domains like the rest